### PR TITLE
spectest: hard-code "spectest" module

### PR DESCRIPTION
**Pull request by gumb0**
Requires https://github.com/wasmx/fizzy/pull/135 and https://github.com/wasmx/fizzy/pull/170

### CODE REVIEW

**Comment by axic** [comment]
Looks good to me but can't merge this before #170.

File: test/spectests/spectests.cpp [hunk 1]
```diff
@@ -16,6 +17,35 @@ namespace
 constexpr auto JsonExtension = ".json";
 constexpr auto UnnamedModule = "_unnamed";
 
+// spectest module details:
+// https://github.com/WebAssembly/spec/issues/1111
+// https://github.com/WebAssembly/spec/blob/e3a3c12f51221daaa901691e674d7ff990029122/interpreter/host/spectest.ml#L33
+// https://github.com/WebAssembly/spec/blob/f85f6e6b70fff9ee0f6ba21ebcdad2183bd67874/test/harness/sync_index.js#L79
+/* wat2wasm
```
Comment (axic) [inline test/spectests/spectests.cpp]: @chfast @gumb0 Hmm, can we make this to be covered by wat2wasm2cpp?

Comment (axic) [reply test/spectests/spectests.cpp]: The regexp seem to cover it.

Comment (axic) [reply test/spectests/spectests.cpp]: Fixed.

File: wat2wasm4cpp.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 FORMAT_TOOL = 'clang-format'
 
 WAT_RE = re.compile(r'/\* wat2wasm(.*)\n([^*]*)\*/', re.MULTILINE)
-WASM_RE = re.compile(r'\s*(?:const )?auto \w+ =\s*from_hex\(\s*"([^;]*)"\);',
+WASM_RE = re.compile(r'\s*(?:const )?auto \w+ =\s*(?:fizzy\:\:)?from_hex\(\s*"([^;]*)"\);',
```
Comment (axic) [inline wat2wasm4cpp.py]: @chfast can you check this? I'm sure about regexp in python.

Comment (chfast) [reply wat2wasm4cpp.py]: This is ok, matches `fizzy::from_hex` additionally to `from_hex`.

Comment (axic) [reply wat2wasm4cpp.py]: Not familiar with the leading `?:` notation, haven't seen it in other regexp. It didn't work without it.

Comment (chfast) [reply wat2wasm4cpp.py]: The `()` create a group - it changes the indexes of other groups. If you add `?:` the group is not capturing and will not get its own index.

**Pull request merged**